app/views/govuk_publishing_components/components/_chart.html.erb

```diff
@@ -0,0 +1,139 @@
+<%
+  add_gem_component_stylesheet("chart")
+  add_gem_component_stylesheet("table")
+  add_gem_component_stylesheet("details")
+  add_gem_component_stylesheet("heading")
+
+  chart_heading ||= nil
+  chart_heading_level ||= 2
+  table_direction ||= "horizontal"
+  h_axis_title ||= nil
+  v_axis_title ||= nil
+  rows ||= []
+  keys ||= []
+  chart_overview ||= nil
+  hide_legend ||= false
+
+  chart_id = "chart-id-#{SecureRandom.hex(4)}"
+  table_id = "table-id-#{SecureRandom.hex(4)}"
+
+  require "chartkick"
+  Chartkick.options[:html] = '<div id="%{id}"><noscript><p class="govuk-body">Our charts are built using JavaScript but all the data is also available in the table below.</p></noscript></div>'
```

Comment:
Also, does this text need editorial approval?

Reply:
Possibly. If it's alright with you I think we'll look at that later when we're reviewing the pages as a whole.